public/js/main.js

```diff
@@ -151,24 +151,31 @@ function redirectToUserPage() {
     // Get the cached user.
     const me = localStorage.myGitHub;
     if (me) {
-        window.location.href = `/?username=${me}`;
+        window.location.href = `./?username=${me}`;
     } else {
-        window.location.href = '/'; // Username not saved, go to main page.
+        window.location.href = './'; // Username not saved, go to main page.
     }
 }
 
 function saveUserPage() {
     // Save username into localStorage. Recall username by visiting /me.
     localStorage.myGitHub = $('[name="username"]').val();
     // Provide some sort of visual feedback. Redirect to that page.
-    window.location.href='/me';
+    window.location.href='./me';
 }
 
 $(document).on('ready', () => {
     // Save is bound to button press
     $(document).on('click', '.saveUser', () => saveUserPage());
     // Works with /me or /me/
-    if (window.location.pathname.startsWith('/me')) {
+    if (window.location.pathname.endsWith('/me')) {
         redirectToUserPage();
     }
+    // if hostname is empty, use location to build the link
+    // needed for reverse proxy setup
+    if (document.getElementById('melink').innerHTML == '/me') {
```

Comment:
This causes console errors for me on the main page where there is no me link, are you able to fix? Also, we have jquery (well zepto) available so you should be able to just do `$('#meLink')`

Reply:
Sorry for that. I missed it.

>  Also, we have jquery (well zepto) available so you should be able to just do $('#meLink')

I have no idea what that means. This is the first time I've done something with js...